### ISSUE TITLE
Chyba v řazení atributů při načtení mineru

### ISSUE THREAD

**Issue by vojir** [KIZI/EasyMiner-MiningUI#95]
Při načtení mineru nedojde k seřazení atributů - jsou seřazeny až při přidání některého z atributů do vzoru pravidla.

![attributes - error](https://cloud.githubusercontent.com/assets/3865942/16780058/40652a62-4875-11e6-9cd3-2f65d1ef2a20.png)

**Comment by PreVaDu** [comment]
Opraveno. Jen to teď nevypadá tak hezky, protože proběhne řazení i s animací po načtení, takže se to chová, jakoby to dělalo něco, co uživatel neovlivní.
Je tato chyba nová, nebo jsme si ji dříve nevšimli? Abych to případně zkusil pořešit na úrovni načítání dat.

**Comment by vojir** [comment]
Asi jsme si toho dřív jen nevšimli. Předpokládám správně, že při výchozím správném seřazení položek k žádné animaci nedojde? (můžu doplnit seřazení seznamu atributů do PHP připravujícího pole s daty)

**Comment by PreVaDu** [comment]
Ano, pokud jsou načteny atributy rovnou seřazené, funkce sice proběhně, ale animace není třeba.

Všiml jsem si ovšem jiné chyby - když skryji atribut a znovu načtu miner, zobrazí se sice správně "Show hidden", ale skrytý atribut je stále viditelný... Pamatuji si, jak jsme ty skryté řešili a myslím, že to bylo v pořádku. Nemohlo se tam celkově něco změnit na logice, co ovlivnilo tyto funkcionality? (řazení, skrývání apod.) Jestli se nezměnil třeba backend, že již nevrací seřazené atributy, nevrací zvlášť hidden apod.
